src/bundle/Security/Http/Authenticator/Passport/TwoFactorPassport.php

```diff
@@ -4,15 +4,18 @@
 
 namespace Scheb\TwoFactorBundle\Security\Http\Authenticator\Passport;
 
+use RuntimeException;
 use Scheb\TwoFactorBundle\Security\Authentication\Token\TwoFactorTokenInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
 use Symfony\Component\Security\Http\Authenticator\Passport\Credentials\CredentialsInterface;
 use Symfony\Component\Security\Http\Authenticator\Passport\PassportInterface;
 use Symfony\Component\Security\Http\Authenticator\Passport\PassportTrait;
+use Symfony\Component\Security\Http\Authenticator\Passport\UserPassportInterface;
 
 /**
  * @final
  */
-class TwoFactorPassport implements PassportInterface
+class TwoFactorPassport implements PassportInterface, UserPassportInterface
```

Comment:
`PassportInterface` is redundant and can be removed, because `UserPassportInterface` is extending `PassportInterface`.